README.md

```diff
@@ -20,8 +20,7 @@ Additional description about the project and its features.
 
 ## Getting Started
 
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+To set up locally on your machine `Git pull` then open the file on your browser.
 
```

Comment:
- [ ] Please remove unnecessary files in our project, for instance you did duplicate files while you were setting up linters
   
![port7](https://user-images.githubusercontent.com/55635977/129166084-3914c8d6-0cb1-424d-bd4a-af3b701b0c89.png)